src/qtism/common/ContentPackageExceptionInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ * @license GPLv2
+ */
+declare(strict_types=1);
+
+namespace qtism\common;
+
+/**
+ * The ContentPackageExceptionInterface intended to help unify the exception handlers behavior
+ */
+interface ContentPackageExceptionInterface
```

Comment:
```suggestion
interface PackageContentExceptionInterface
```

Reply:
@wazelin Should https://www.php-fig.org/bylaws/psr-naming-conventions/ be respected?

Reply:
What do you think about using the lib name in the base exception class just like [Guzzle does](https://github.com/guzzle/guzzle/blob/master/src/Exception/GuzzleException.php)?

Something like: `QtiSdkException`